### PR TITLE
refactor: optimize ovn command when error exists

### PR DESCRIPTION
**Pull request by xieyanker**

### CODE REVIEW

File: pkg/ovs/ovn-ic-nbctl.go [hunk 1]
```diff
@@ -14,9 +14,8 @@ func (c Client) ovnIcNbCommand(cmdArgs ...string) (string, error) {
 	raw, err := exec.Command(OVNIcNbCtl, cmdArgs...).CombinedOutput()
 	elapsed := float64((time.Since(start)) / time.Millisecond)
 	klog.V(4).Infof("%s command %s in %vms", OVNIcNbCtl, strings.Join(cmdArgs, " "), elapsed)
-	if err != nil || elapsed > 500 {
-		klog.Warning("ovn-ic-nbctl command error or took too long")
-		klog.Warningf("%s %s in %vms", OVNIcNbCtl, strings.Join(cmdArgs, " "), elapsed)
+	if elapsed > 500 {
+		klog.Warning("ovn-ic-nbctl command took too long")
```
Comment (oilbeater) [inline pkg/ovs/ovn-ic-nbctl.go:18]: we still need to log the slow query here

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: pkg/ovs/ovn-ic-nbctl.go (hunk 1 above)
Comment (xieyanker) [reply pkg/ovs/ovn-ic-nbctl.go]: I pushed the code again, is it okay now?

**Review by oilbeater — APPROVED** [review]

**Comment by oilbeater** [comment]
Thanks! @xieyanker

**Pull request merged**